frontend/src/metabase/search/components/SearchResult.jsx

```diff
@@ -179,34 +253,83 @@ function DashboardResult({ dashboard, options }) {
 function QuestionResult({ question, options }) {
   return (
     <ResultLink to={question.getUrl()} compact={options.compact}>
-      <Flex align="center">
-        <ItemIcon item={question} />
+      <Flex align="start">
+        <ItemIcon item={question} type="question" />
         <Box>
           <Title>{question.name}</Title>
-          {formatCollection(question.getCollection())}
+          <Text>{jt`Saved question in ${formatCollection(
```

Comment:
The biggest thing that differed between all these types ended up just being the "context" block, and for several types all that really changes is the "model" text (e.g. Saved question, dashboard, etc). I think I'll look into trying to separate that out and keep the overall wrapper as similar as possible between all the types now that it's all shaken out.

Reply:
I think my initial vision was for the context block to be a generic wrapper over what would normally be description. IMHO. If we treat description as context, we'll get the bolding for free. Something like:

![image](https://user-images.githubusercontent.com/784417/111005405-5d3f8b80-8350-11eb-9498-0c958c1c4fb3.png)